src/testing/models.ts

```diff
@@ -8,6 +10,7 @@ export interface Threshold {
 export interface Evaluation {
   score: number;
   threshold?: Threshold;
+  metadata?: Record<string, unknown>;
```

Comment:
also i'd update the type to `ArbitraryProperties` which is `Record<string, any>` b/c it would allow someone to type their metadata w/ an interface

https://github.com/autoblocksai/javascript-sdk/blob/1afd0740eb56653af87a47e498c8b4a02a63b127/src/types.ts#L3-L6

see https://stackoverflow.com/q/65799316